README.md

```diff
@@ -378,19 +418,19 @@ Occasionally, it can be helpful for our subscribers to test CSS changes before t
 
 To publish prerelease versions:
 
-* First, be sure that you're working on a branch other than `main`.
-* Once your change(s) are ready to be committed, be aware of the severity of the change(s), and be sure to author your commit message so that Lerna understands how to increase the version number(s) of the affected components.
-* Once your changes are committed, you **must** build the affected package(s) locally **before** publishing them to npm. An npm task for cleaning, building, and beta publishing is available, and it can be run via the following command: `yarn release:beta-from-package`. This command will perform a full `clean` (via the `clean` task), a full `build` (via the `build` task), and will attempt to bump the version numbers in the affected package(s) (via `lerna publish --conventional-prerelease --preid beta --pre-dist-tag beta --no-private`).
-* Depending on the severity of your change(s), and before publishing to npm, Lerna should show a preview of the affected package version numbers that look something like: `@spectrum-css/tag: 3.3.8 => 3.3.9-beta.0`. Additionally, at this time, Lerna will ask if you would like to continue with publishing the changes or cancel.
-* Selecting `y` to publish will publish the affected package(s) to npm.
+- First, be sure that you're working on a branch other than `main`.
```

Comment:
Some of this is because I ran it through prettier manually to catch any stray typos or minor issues